src/pages/homePage/homePage.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './homePage.scss';
+import MovieCard from '../../components/movieCard/movieCard';
+
+const HomePage = () => {
```

Comment:
How about the Home Component, what's the difference between this and that component??